Keycloak.Migrator/Keycloak.Migrator.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
 	<!--<TargetFramework>net6.0</TargetFramework>-->
-	<TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+	<TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Reason for the target frameworks being downgraded?

Reply:
Mistake when I was testing some things.